### PR TITLE
Add Intercom sdk & Update Readme for Customers

### PR DESCRIPTION
**Pull request by devintyler**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,9 @@ var Syncano = require("syncano");
 ```
 
 More info about the library can be found [here](https://github.com/Syncano/syncano-js-lib).
+
+# Adding Packages
+
+To add a package, please create a Pull Request from a fork or branch into `master` after adding the package with a specific version number to [this file (nodejs/files/package.json.v10)](https://github.com/Syncano/nodejs-codebox/blob/intercom-sdk/nodejs/files/package.json.v10).
+
+
```
Comment (kotchaosu) [inline README.md]: remove spare lines

File: README.md [hunk 2]
```diff
@@ -29,3 +29,9 @@ var Syncano = require("syncano");
 ```
 
 More info about the library can be found [here](https://github.com/Syncano/syncano-js-lib).
+
+# Adding Packages
+
+To add a package, please create a Pull Request from a fork or branch into `master` after adding the package with a specific version number to [this file (nodejs/files/package.json.v10)](https://github.com/Syncano/nodejs-codebox/blob/intercom-sdk/nodejs/files/package.json.v10).
```
Comment (kotchaosu) [inline README.md]: It's always a branch.

Comment (devintyler) [reply README.md]: spare lines removed, will edit text now.

Comment (kotchaosu) [reply README.md]: Sorry for making myself not clear - even if it's fork it's branch. Leave `Pull Request into master` - should be clear enough.

**Review by kotchaosu — APPROVED** [review]
Small suggestion. You can merge it. I'll deploy it to production tomorrow.

**Pull request merged**